src/vivarium/component.py

```diff
@@ -156,6 +156,28 @@ def name(self) -> str:
 
         return self._name
 
+    @property
+    def population_view(self) -> PopulationView:
+        """Provides the PopulationView for this component.
+
+        Returns
+        -------
+        PopulationView
```

Comment:
I thought I didn't need to specify this return type here since the method is typed, but it fails if I don't.

Reply:
You shouldn't have to declare it here as long as you're using the sphinx_autodoc_typehints extension (which we are), but I have seen this before as well.